src/bin/shuffle.rs

```diff
@@ -11,6 +11,7 @@ fn main() {
     animations.push(|| Box::new(Rainbow::default()));
     animations.push(|| Box::new(RandomWalkers::default()));
     animations.push(|| Box::new(GOL::default()));
+    animations.push(|| Box::new(QrCode::new("https://fckaf.de/p6K", (5, 6))));
```

Comment:
```suggestion
    animations.push(|| Box::new(QrCode::new("https://github.com/991jo/asciimation", (5, 6))));
```

I think we should use the direct link here, so that the QR code does not break when the URL shortener does not work anymore. This results in a slightly larger QR code, but that should be fine.